discovery-infra/test_infra/helper_classes/cluster.py

```diff
@@ -515,3 +551,18 @@ def wait_for_cluster_to_be_in_installing_status(self):
             cluster_id=self.id,
             statuses=[consts.ClusterStatus.INSTALLING]
         )
+
+    def get_events(self, host_id=''):
+        return self.api_client.get_events(cluster_id=self.id, host_id=host_id)
+
+    def verify_events(self, event_to_find, params_list, host_id='', fail_on_error=False):
```

Comment:
where your using this fail_on_error?
and what is param_list? can you give an example?

Reply:
Can you please replace verify_events with verify_event or find_event?

Reply:
params_list=[]

Reply:
Done, please review